spec/test/collections/test_silva.py

```diff
@@ -0,0 +1,267 @@
+import re
+from yaml import safe_load
+from jsonschema.exceptions import ValidationError
+from relation_engine_server.utils.json_validation import run_validator
+import os
+import unittest
+
+cwd = os.path.dirname(os.path.abspath(__file__))
+yaml_drpth = os.path.join(cwd, '../../collections/silva')
+node_yaml_flpth = os.path.join(yaml_drpth, 'silva_taxon.yaml')
+edge_yaml_flpth = os.path.join(yaml_drpth, 'silva_child_of_taxon.yaml')
+
+
+class SILVATreeJSONSchemaTest(unittest.TestCase):
+    '''
+    Test the API of the nodes and edges representing SILVA taxonomy tree
+    All information is from SILVA (arb-silva.de)
+    See their documentation for more details
+    '''
+
+    @classmethod
+    def setUpClass(cls):
+        with open(node_yaml_flpth) as fh:
+            cls.schema_node = safe_load(fh)['schema']
+        with open(edge_yaml_flpth) as fh:
+            cls.schema_edge = safe_load(fh)['schema']
```

Comment:
The validator will do all this for you. Specify the schema file and where in the schema you want to validate against:

````py
run_validator(
   schema_file='/path/to/schema/file.yaml',
   data={'this': 'that'},
   validate_at='/schema',
)
````

If you're going to be doing a lot of validation against the same schema, you can load it up in advance using [`get_schema_validator`](https://github.com/kbase/relation_engine/blob/develop/relation_engine_server/utils/json_validation.py#L42) to get a validator instance:
````py
validator = get_schema_validator(
   schema_file='/path/to/schema/file.yaml',
   validate_at='/schema'
)
validator.is_valid(data_structure)
# alternative way to call the function
validator.validate(data_structure_2)
````